tbot_contrib/swupdate/swupdate_script.py

```diff
@@ -0,0 +1,97 @@
+import asyncio
+import json
+import requests
+import websockets
+import argparse
+
+
+class SWUpdater:
```

Comment:
I feel like a class is overkill for this small script.  I'd rather stuff the contents of the `update()` method into `main()` and define `async def wait_update_finished()` as a top-level function.  Or was there a specific reason why you added a class?